packages/ag-charts-enterprise/src/features/popover/popover.css

```diff
@@ -0,0 +1,33 @@
+.ag-charts-popover {
+    position: absolute;
+    display: grid;
+    grid: auto-flow auto / 1fr;
+    column-gap: 12px;
+    min-width: 200px;
+    background: var(--ag-charts-popover-background-color);
+    border: var(--ag-charts-popover-border);
+    border-radius: 4px;
+    padding: 6px 0;
+    overflow: hidden;
+    /* Needs to beat .ag-crosshair-label */
+    z-index: 99999;
```

Comment:
This should be a variable, like the `--ag-charts-layer-crosshair` etc values

Reply:
Just removed it since we remove the crosshairs now